src/Events/ScoutViewEngineDecorator.php

```diff
@@ -0,0 +1,41 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Scoutapm\Laravel\Events;
+
+use Illuminate\Contracts\View\Engine;
+use Scoutapm\Agent;
+
+final class ScoutViewEngineDecorator implements Engine
+{
+    /** @var Engine */
+    private $realEngine;
+
+    /** @var Agent */
+    private $agent;
+
+    public function __construct(Engine $realEngine, Agent $agent)
+    {
+        $this->realEngine = $realEngine;
+        $this->agent = $agent;
+    }
+
+    /**
+     * Get the evaluated contents of the view.
+     *
+     * @param string $path
+     * @param array $data
+     * @return string
+     */
+    public function get($path, array $data = [])
+    {
+        $this->agent->startSpan('viewEngine');
```

Comment:
Happy to take guidance on what this span should be named, according to other Scout conventions...

Reply:
`View/file path snippet` ideally. So something like `View/users/index`